### DISCUSSION TITLE
Pause Generated \ Library Scans

### DISCUSSION THREAD

**Discussion by hydranmenace** [stashapp/stash#2845] [Ideas]
I don't know if this is the right place for requests but fwiw I would really like to have the ability to pause the Generated Content and Library scans. You can cancel it, clearly, but pausing would be nice. I just ran a new generation process that took the better part of 2 days which I have to restart now. My unraid box rebooted during a storm while the scan was in progress (main PC stayed on). If I could have paused and waited until the server came back up I wouldn't have to start over. Also, while running scans and generation most other intensive processes like gaming really take a hit. Pausing to game and resuming at night when I'm not otherwise using my desktop would be very useful.

Also, when scraping actresses and actors I would really like the ability to say, scrape with StashDB but have multiple images to choose from for the individual in question. As it is I generally have to scrape with StashDB, FreeOnes, Babepedia, and TheNude until I find an image I like which takes a lot of time.

Apologies if this isn't the right place. Seemed reasonable though.

**Comment by scruffynerf** [comment]
You can stop them, and they will resume just fine.  You don't have to restart over.

As for "scrape with StashDB but have multiple images to choose from for the individual in question"  You do, not sure why you don't see that: when it loads the performer, you can choose from any of the available images, and pick one.

**Comment by ALonelyJuicebox** [comment]
What Scruffy said-- Stash performs generation tasks on the media that actually needs things (sprites, previews, etc) generated for it. If you stop a generate task and start it over again, you don't start from 0, you start from wherever the task left off. 🙂

**Reply by jdrch** [reply]
Dumb question: does this include the situation in which the initial scan after setup is canceled and then a new scan is started? I wound up having to do that because the initial scan was taking too long (only 1 thread initially assigned).

**Reply by feederbox826** [reply]
Yes, it will continue normally

**Reply by jdrch** [reply]
> Yes, it will continue normally

Excellent, thank you!
